news/2024/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks.md

```diff
@@ -0,0 +1,308 @@
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/4568537">Predominador</a></p>
+
+For now, I have 2 concrete plans: to reach 101k SS, because it means reach 100k SS Silver; and to reach 1 Trillion ranked score, which is actually something I never did grind directly, it just happened to me gaining a lot due consequences of SS farming... but I think it'd be cool to reach 1 trillion (for now just syu did it), and I am already close to do both, actually. However, for now, I am prioritizing rest.
+After that, I will probably return to try to get some medals I still don't have, osu!catch exclusive maps completionist, and also improve my skill not only in osu!catch, but other game modes too. Other specific goals I will think more about in the future!
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Finally, do you have anything else that you’d like to mention?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/4568537" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-Predominador.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/4568537">Predominador</a></p>
+
+I want to say that thanks to osu!, I had the opportunity to know some artists that their songs are very important to me, such as Mili, Porter Robinson, Aimer, Kenshi Yonezu, Camellia, Nekomata Master, and many others! 
+
+Thanks to my friends and everyone that support me and have some consideration! And I hope I somehow inspired some of you to pursue your goals, or maybe even play osu!catch!
+
+Even though we all have moments of stress and frustration here, it's always good to remember to prioritize having fun! This is a game, after all.
+
+And thanks for the opportunity to be interviewed on The Followpoint, it was really an honor! I really hope you all enjoyed reading this interview! Now let's enjoy game õ/
+
+</div>
+
+---
+
+Many thanks to Predominador for agreeing to this interview and their thoughtful answers! And of course, thank you so much to every one of you who support this series, it means the world to us!
+
+Please look forward to a new, ***very special interview*** coming out in the next three weeks, and make sure to keep an eye on the news feed! You may also follow [@The_Followpoint on Twitter](https://twitter.com/The_Followpoint) to see more updates and question submissions for future interviews.
```

Comment:
Sending this as a suggestion to make sure that's fine to do this. @Hiviexd 
```suggestion
Please look forward to a new, ***very special interview*** coming out in the next three weeks, and make sure to keep an eye on the news feed! You may also follow [@The_Followpoint on Twitter](https://twitter.com/The_Followpoint) (We also have a [Bluesky account](https://bsky.app/profile/thefollowpoint.megamix.dev) now!) to see more updates and question submissions for future interviews.
```

Reply:
sure